julia.html.markdown

```diff
@@ -478,67 +478,67 @@ catch e
 end
 
 # You can define functions that take keyword arguments
-function keyword_args(;k1=4, name2="hello")  # note the ;
+function keywordArgs(;k1=4, name2="hello")  # note the ;
     return Dict("k1" => k1, "name2" => name2)
 end
-# => keyword_args (generic function with 1 method)
+# => keywordArgs (generic function with 1 method)
 
-keyword_args(name2="ness")  # => ["name2"=>"ness", "k1"=>4]
-keyword_args(k1="mine")     # => ["name2"=>"hello", "k1"=>"mine"]
-keyword_args()              # => ["name2"=>"hello", "k1"=>4]
+keywordArgs(name2="ness")  # => ["name2"=>"ness", "k1"=>4]
+keywordArgs(k1="mine")     # => ["name2"=>"hello", "k1"=>"mine"]
+keywordArgs()              # => ["name2"=>"hello", "k1"=>4]
 
 # You can combine all kinds of arguments in the same function
-function all_the_args(normal_arg, optional_positional_arg=2; keyword_arg="foo")
-    println("normal arg: $normal_arg")
-    println("optional arg: $optional_positional_arg")
-    println("keyword arg: $keyword_arg")
+function allTheArgs(normalArg, optionalPositionalArg=2; keywordArg="foo")
+    println("normal arg: $normalArg")
+    println("optional arg: $optionalPositionalArg")
+    println("keyword arg: $keywordArg")
 end
-# => all_the_args (generic function with 2 methods)
+# => allTheArgs (generic function with 2 methods)
 
-all_the_args(1, 3, keyword_arg=4)
+allAheArgs(1, 3, keywordArg=4)
 # => normal arg: 1
 # => optional arg: 3
 # => keyword arg: 4
 
 # Julia has first class functions
-function create_adder(x)
+function createAdder(x)
     adder = function (y)
         return x + y
     end
     return adder
 end
-# => create_adder (generic function with 1 method)
+# => createAdder (generic function with 1 method)
 
 # This is "stabby lambda syntax" for creating anonymous functions
 (x -> x > 2)(3)  # => true
 
-# This function is identical to create_adder implementation above.
-function create_adder(x)
+# This function is identical to createAdder implementation above.
+function createAdder(x)
     y -> x + y
 end
-# => create_adder (generic function with 1 method)
+# => createAdder (generic function with 1 method)
 
 # You can also name the internal function, if you want
-function create_adder(x)
+function createAdder(x)
     function adder(y)
         x + y
     end
     adder
 end
-# => create_adder (generic function with 1 method)
+# => createAdder (generic function with 1 method)
 
-add_10 = create_adder(10) # => (::getfield(Main, Symbol("#adder#11")){Int64}) 
+add10 = createAdder(10) # => (::getfield(Main, Symbol("#adder#11")){Int64}) 
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194